app/models/client_opportunity_match.rb

```diff
@@ -767,6 +776,10 @@ def succeeded!(touch_referral_event: true, user: nil)
         client.make_unavailable_in(match_route: route, user: user, match: self, reason: UnavailableAsCandidateFor::SUCCESSFUL_MATCH_TEXT)
       end
 
+      match_route.routes_parked_on_successful_match.reject(&:empty?).each do |park_route|
```

Comment:
```suggestion
      # Park client on any other routes where this route is set to block matching when the client is successful on this route
      match_route.routes_parked_on_successful_match.reject(&:empty?).each do |park_route|
```

Reply:
This has been included in the lates commit